.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on: [pull_request]
+
+jobs:
+  tests: 
+    name: Tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v3
+
+      - name: Docker Compose
+        run: |
+            docker-compose up -d
+            sleep 5
+
+      - name: Composer Install
+        run: |
+            docker run --rm --volume "$(pwd)":/app composer install --ignore-platform-reqs --optimize-autoloader --no-plugins --no-scripts --prefer-dist
+      
+      - name: Run Tests
+        run: |                
+            docker ps
+            docker-compose exec -T php8 vendor/bin/psalm --show-info=true
```

Comment:
```suggestion
            
```
We don't need this as we already have a linter workflow for that

Reply:
@2002Bishwajeet 
Isn't psalm an analyser??
We are already using phpstan analyser (https://github.com/utopia-php/cache/blob/master/composer.json#L9)

Reply:
Yep, it is. But we don't want to use this in a test workflow

Reply:
@2002Bishwajeet Done